### ISSUE TITLE
training on turkish dataset

### ISSUE THREAD

**Issue by abdelhamidfares** [facebookresearch/wav2letter#650]
hello i am using common voice dataset for Turkish to train that is link for dataset after prepared to work with wav2letter https://drive.google.com/open?id=1gEJt0SC62jQtjv-1CzIem4CWY9fh8fSD.
i used your model in tutorial for training that is train.cfg file 
[train.cfg.txt](https://github.com/facebookresearch/wav2letter/files/4620665/train.cfg.txt)
number of epoch used 180. loss for train is decreasing and WER for train is decreasing but for test still very high.
that is the log file for training
[001_log.txt](https://github.com/facebookresearch/wav2letter/files/4620698/001_log.txt)
how can i edit train configure file to have better WER for test and get better loss
thanks for your help

**Comment by padentomasello** [comment]
Hi @abdelhamidfares, it sounds like your model is overfitting. Looking at the common voice dataset for Turkish, it looks like it's only 13 hours, (where as English is over 1000), so this is not too surprising. 

I would encourage you to look into regularization techniques such as dropout, weight decay, and data augmentation, which may help the problem.

**Comment by padentomasello** [comment]
Hi @abdelhamidfares. Closing for now. Hope you have some luck with some of the techniques mentioned above.
